test/server/cards/01_SOR/events/TheEmperorsLegion.spec.ts

```diff
@@ -0,0 +1,55 @@
+describe('The Emprerors Legion', function () {
+    integration(function (contextRef) {
+        it('should return card to player hand from a discard pile', function () {
+            contextRef.setupTest({
+                phase: 'action',
+                player1: {
+                    hand: ['the-emperors-legion'],
+                    groundArena: ['pyke-sentinel', 'seasoned-shoretrooper', 'blizzard-assault-atat'],
+                    spaceArena: ['cartel-spacer'],
+                    discard: ['keep-fighting', 'green-squadron-awing', 'disarm']
+                },
+                player2: {
+                    hand: ['superlaser-blast'],
```

Comment:
Can we add some units for player2 and that get defeated and confirm that they are not returned by Emperor's Legion